todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -61,7 +65,14 @@ const createTodoItem = createWidgetBase
 							state: { classes: [ 'toggle' ], checked }
 						}),
 						d(createLabel, {
-							listeners: { dblclick: () => { todoEdit.do(state); } },
+							listeners: {
+								dblclick: () => { todoEdit.do(state); },
+								keypress: (event: KeyboardEvent) => {
+									if (event.which === 13) {
```

Comment:
@agubler I'll move the event.which to userActions. If there's no objection, I think it would also be a good idea to use event.which for all key checking in userActions, since it normalizes keyCode and charCode. I agree we should probably add some key event util for KeyboardEvent.key, but probably not in todo MVC :P